terrautils/brapi.py

```diff
@@ -0,0 +1,130 @@
+"""brapi
+
+This module provides wrappers to BRAPI API for getting and posting data.
+"""
+
+import os
+
+import requests
+import urllib
+
+BRAPI_URL="https://brapi.workbench.terraref.org/brapi"
+BRAPI_VERSION="v1"
+
+
+def brapi_get(path='',request_params=None):
+    brapi_url = os.environ.get('BRAPI_URL', BRAPI_URL)
+    path = 'brapi/'+path
+    request_url = urllib.parse.urljoin(brapi_url, path)
```

Comment:
```>>> from terrautils.betydb import get_sites      
>>> s = get_sites('2017-06-06')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/terrautils/betydb.py", line 243, in get_sites
    exp_site_cultivar_map = brapi.get_site_id_cultivar_info_map(exp['id'])
  File "/usr/local/lib/python2.7/dist-packages/terrautils/brapi.py", line 111, in get_site_id_cultivar_info_map
    layouts = get_brapi_study_layouts(studyDbId)
  File "/usr/local/lib/python2.7/dist-packages/terrautils/brapi.py", line 67, in get_brapi_study_layouts
    data = brapi_get(path=current_path)
  File "/usr/local/lib/python2.7/dist-packages/terrautils/brapi.py", line 18, in brapi_get
    request_url = urllib.parse.urljoin(brapi_url, path)
AttributeError: 'module' object has no attribute 'parse'```

Reply:
resolved in https://github.com/terraref/terrautils/pull/71#issuecomment-499963763